libs/ui/src/store/wallet/wallet.selectors.ts

```diff
@@ -164,8 +164,10 @@ export const useVisibleAccountTokensAndGasTokenSelector = (): Token[] => {
 
 export const useCollectiblesSelector = () => {
   const assets = useAccountAssetsSelector();
+  const { rpcUrl, chainId } = useSelectedNetworkSelector();
+  const publicKeyHash = useSelectedAccountPublicKeyHashSelector();
 
-  return useMemo(() => assets.filter(token => isCollectible(token)), [assets]);
+  return useMemo(() => assets.filter(token => isCollectible(token)), [assets, rpcUrl, chainId, publicKeyHash]);
```

Comment:
looks like `useAccountAssetsSelector` works wrong.  
If `assets` depends on `rpcUrl, chainId, publicKeyHash` - it should return new value when those are changes.  
in your case - this bug will still work in other places where `assets` list do not updates when it's needed